content/tutorials/single-page-tutorials/wallet-setup.md

```diff
@@ -0,0 +1,431 @@
+# Setup testnet address and receive funds from faucet
+
+In this tutorial you will create a new mnemonic seed phrase in order to setup a Stronghold account with the [NodeJS binding of wallet.rs](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/nodejs). Afterwards you will generate a new address, request funds from the Shimmer testnet faucet and check your account balance.
+
+## Prerequisites
+
+- Development Environment and Libraries
+  - Installed [Node.js](https://nodejs.org/en/)
+
+***
+
+## Setup
+
+### Prepare development environment
+
+Create a new folder for the tutorial in a location of your choice and navigate to it:
+
+```bash
+mkdir wallet-setup
+cd wallet-setup
+```
+
+Run the Node.js initializer to configure the project:
+
+```bash
+npm init --yes
+```
+
+Overwrite the `package.json` file with the following content:
+
+```yaml
+{
+  "name": "wallet-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "create-mnemonic.js",
+  "dependencies": {
+    "@iota/wallet": "2.0.2-alpha.21",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1",
+    "node-fetch": "^2.6.7"
+  },
+
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+
+Install new dependencies:
+:::note
+
+Installation might take a while, if there is no prebuilt wallet NPM package for your combination of OS and NodeJS version.
+
+:::
+```bash
+npm install
+```
+
+***
+
+### Prepare network configuration
+
+Create a new file `networkConfig.js` and add the following code:
+
+```javascript
+var networkConfig = {};
+
+// Shimmer Beta network configuration
+networkConfig.node = 'https://api.testnet.shimmer.network';
+networkConfig.faucetWebsite = 'https://faucet.testnet.shimmer.network';
+networkConfig.faucetApi = 'https://faucet.testnet.shimmer.network/api/enqueue';
+networkConfig.explorer = 'https://explorer.shimmer.network/testnet';
+
+module.exports = { networkConfig };
+```
+
+***
+
+### Prepare environment variables

```

Comment:
```suggestion
### Prepare the Environment Variables
```

Reply:
should we add a warning to indicate that using dotenv isn't safe in prod environments?